workshop-steps/workshop.html

```diff
@@ -49,258 +49,474 @@
 </div>
 
 <div class="container">
-  <h1>Playground</h1>
+  <h1>Installation Steps</h1>
+  To do this workshop, it's necessary to have some components installed in the local machine.
+  Depending on your environment, you can find detailed installation instructions below:
```

Comment:
Can you add two links to the relevant sections here? Like:

```html
<ul>
  <li><a href="#_restricted-internet-environment">Restricted Internet Environment</a</li>
  <li><a href="#_unrestricted-internert-environment">Unrestricted Internet Environment</a</li>
</ul>
```

Reply:
Ok, I'll add those inside installation steps, should be quick.

I do think we need a full table of contents at the top though as well as this, what do you think?

Reply:
If eventually we move the content to Asciidoc, then the TOC will come for free. In the meantime, I'm not really keen on maintaining one manually :-)